### PR TITLE
Surajsahani patch 1

### PR DESCRIPTION
**Pull request by surajsahani**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by surajsahani** [comment]
Hi, @saarques   Hope you are doing well :)
Can you please help me reviewing my pr. it would be very helpful I have removed unwanted files and fixed image size.

File: _data/surajsahani/surajsahani.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: SURAJ SAHANI
+institution: Rawal Institute Of Engineering And Technology
+profile_pic: surajsahani.png
```
Comment (mitulsavani) [inline _data/surajsahani/surajsahani.md:4]: The image path name seems incorrect, you might want to change `surajsahani.png` to `surajsahani.jpg`

Comment (surajsahani) [reply _data/surajsahani/surajsahani.md]: Thanks, mitulsavani I have to change the file requested can you review again. thank you.

**Comment by surajsahani** [comment]
> 1. Checklist complete--(Okay)
> 2. Files modified--2(Okay)
> 3. Profile picture checked--(Okay)
> 
> Looks good to me :shipit:
> 
> > Congratulation on your graduation confetti_ball

Thanks man

**Comment by saarques** [comment]
All's cool! :)

**Pull request merged**